pcdsdevices/sequencer.py

```diff
@@ -189,3 +199,131 @@ def stop(self):
         """Stop the EventSequencer"""
         logger.debug("Stopping the EventSequencer")
         self.play_control.put(0)
+
+
+class SequenceLine(Device):
+    """Sub-class for event sequencer line."""
+
+    ec = FCpt(EpicsSignal, '{self.prefix}:EC_{self._ID}:{self._line}')
+    bd = FCpt(EpicsSignal, '{self.prefix}:BD_{self._ID}:{self._line}')
+    fd = FCpt(EpicsSignal, '{self.prefix}:FD_{self._ID}:{self._line}')
+    bc = FCpt(EpicsSignal, '{self.prefix}:BC_{self._ID}:{self._line}')
+
+    def __init__(self, prefix, hutch_id=None, line=None, **kwargs):
+
+        self._ID = hutch_id
+        self._line = line
+
+        super().__init__(prefix, **kwargs)
+
+    def read(self):
```

Comment:
This breaks the `bluesky` API by redefining `read`. I don't think this device will ever be read by `bluesky`, but if it does we should be careful here. Even if we don't, I suspect this could get confusing.